test/Rings/PBWAlgebraQuo.jl

```diff
@@ -1,3 +1,11 @@
+import Oscar.AbstractAlgebra
+include(joinpath(pathof(AbstractAlgebra), "..", "..", "test", "Rings-conformance-tests.jl"))
```

Comment:
Doesn't including this multiple times produce warnings about stuff being redefined? If so I guess we could make the `include` conditional, wrapping `if !isdefined(Main, test_Field_interface)` around the include might do the job

Reply:
That should only happen during precompilation, not during runtime. But nonetheless its good practice to guard here. Added now